config/software/stunnel.rb

```diff
@@ -54,16 +54,21 @@
     env["WIN32_SSL_DIR_PATCHED"] = "#{install_dir}/embedded"
 
     if windows_arch_i386?
```

Comment:
Just a small style thing, might be more clear to just use the conditional (or ternary operator) to set the `windows_build_type` var only...something like:

```ruby
windows_build_type = windows_arch_i386? ? "mingw" : "mingw64"
make windows_build_type, env: env, cwd: "#{project_dir}/src"
```

We should also begin to track how much we need code to determine code to determine `mingw`/`mingw64`...when you hit three times doing this in software definitions we should expose a first class helper in Omnibus (like [we have for `windows_arch_i386?` and `fips_mode?`](https://github.com/chef/omnibus/blob/master/lib/omnibus/sugarable.rb#L60-L74)).